letta/persistence_manager.py

```diff
@@ -42,7 +30,6 @@ def update_memory(self, new_memory):
 class LocalStateManager(PersistenceManager):
```

Comment:
Could we just remove this entire class, and have the agent access `EmbeddingArchivalMemory` directly? This class doesn't seem to be doing anything anymore

Reply:
done